rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2433,6 +2435,35 @@ void voteFederationChange_addFederatorPublicKey_returnsExpectedPendingFederation
 
         }
 
+        @Test
+        void voteFederationChange_addWrongFederatorPublicKey_returnsGenericResponseCode() {
```

Comment:
nit, but since you are naming the variable `invalidPublicKey`, i would say `_whenAddingInvalidFederatorPublicKey`

Reply:
and again, `GenericErrorResponseCode`

Reply:
Updated.